documentation/docs/steps/deployIntegrationArtifact.md

```diff
@@ -1,7 +1,6 @@
 # ${docGenStepName}
 
-${docGenDescription}
-
+## ${docGenDescription}
 With this step you can deploy a integration flow artifact in to SAP Cloud Platform integration runtime using OData API.
 
 Learn more about the SAP Cloud Integration remote API for deploying an integration artifact [here](https://help.sap.com/viewer/368c481cd6954bdfa5d0435479fd4eaf/Cloud/en-US/08632076a1114bc1b6a1ecafef8f0178.html).
```

Comment:
I believe the result will be the description of the metadata file followed by this text, so:
```
Deploy a CPI integration flow
With this step you can deploy a integration flow artifact in to SAP Cloud Platform integration runtime using OData API.

Learn more about the SAP Cloud Integration remote API for deploying an integration artifact [here](https://help.sap.com/viewer/368c481cd6954bdfa5d0435479fd4eaf/Cloud/en-US/08632076a1114bc1b6a1ecafef8f0178.html).
```